website/src/views/home.js

```diff
@@ -26,6 +31,7 @@ function Home() {
         api.get('/grades').then((res) => {
             if(mounted){
```

Comment:
If if depends on a user being logged in, we should ensure there is a token before making the request.
```suggestion
            if(mounted && localStorage.getItem('token'){
```

Reply:
Resolved in commit #d3e90